install-dev/theme/js/process.js

```diff
@@ -177,7 +177,7 @@ function install_error(step, errors)
 			list_errors = [];
 			list_errors[0] = errors;
 		} else if ($.type(list_errors) == 'array') {
-			list_errors = list_errors[0];
+			list_errors = [list_errors[0]];
```

Comment:
Aren't we missing additional error messages by keeping only the first row?

Reply:
It was like that since a while... but currently throw a JS error on the each below.